File: Beginner_Projects/Patterns/Alphabats/C++/Q.cpp
#include <iostream>
#include <conio.h>
using namespace std;

int main()
{
    for (int row = 0; row < 6; row++)
    {
        for (int col = 0; col < 7; col++)
        {
            if (((row * col == 0 && (col + row != 0) && col != 6 && row < 4)) || (col == 6 && row != 0 && row != 4) || (row == 4 && col != 0 && col < 6) || (row == 3 && col == 4))

            {
                cout << "*";
            }

            else
            {
                cout << " ";
            }
        }
        cout << endl;
    }
    getch();
}

File: Beginner_Projects/Patterns/Alphabats/Python/Q.py
for row in range(6):
    for col in range(8):
        if (((row * col == 0 and (col + row != 0) and col < 6 and row < 4)) or (col == 6 and row != 0 and row <4) or (row == 4 and col != 0 and col!=7) or (row == 3 and col == 5)) or (row==5 and col==7):
            print("*", end=" ")
        else:
            print(end="  ")
    print()
input()
